types/redux-form/index.d.ts

```diff
@@ -13,7 +13,7 @@
 //                 Kamil Wojcik <https://github.com/smifun>
 //                 Mohamed Shaaban <https://github.com/mshaaban088>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.0
+// TypeScript Version: 3.3
```

Comment:
Why did the TS version need to be bumped?

Reply:
I'm unclear on the purpose of this header (first time contributing, sorry). The test file passes up to and including TS 3.1, but fails on 3.2+ without these changes. What should this header reflect?

Reply:
No worries, it can be a little confusing. There's some documentation on the readme: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-typescript-21-or-above

This value is the minimum TS version you need in order to be able to use the typings.

_With_ your current changes, do the tests still pass for TS 3.0, 3.1 and 3.2? If so you don't need to update it. In fact you typically only need to bump this version if the CI tests warn you (they test against multiple versions of TS and check for discrepancies)

Reply:
With the changes they pass for all versions of TS that CI tests against, so I'll revert it. Thanks for the pointer 👍